app/services/marketplace_service/community.rb

```diff
@@ -36,6 +36,12 @@ def payment_type(community_id)
           .or_else(nil)
       end
 
+      def shipping_enabled?(community_id)
+        Maybe(MarketplaceSettings.where(community_id: community_id).first)
+          .map { |settings| settings.shipping_enabled }
+          .or_else(nil)
+      end
```

Comment:
since this is temporary it could only exist in controller instead of in deprecated service